release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -196,6 +196,18 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-operators"]
+=== Operators
+
+[id="ocp-4-6-cloud-credential-operator-mode-support"]
+==== Support for Cloud Credential Operator modes
+
+The xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator (CCO)]
+can now be configured to operate in the following modes: `Mint`, `Passthrough`,
```

Comment:
I would suggest adding some kind of mention that the original default behavior (credentialsMode: "") is still there and is still the default mode. Something along these lines:
```suggestion
can now be explicitly configured to operate in the following modes (in addition to the previous and still default mode of operation): `Mint`, `Passthrough`,
```